spec/unit/functions/unwrap_spec.rb

```diff
@@ -15,6 +15,14 @@
     expect(eval_and_collect_notices(code)).to eq(['unwrapped value is 12345'])
   end
 
+  it 'just returns a non-sensitive value' do
+    code = <<-CODE
+      $non_sensitive = "12345"
+      notice("value is still ${non_sensitive}")
```

Comment:
Shouldn't this test call `unwrap` on the variable?

Reply:
👍 Yes, you're absolutely right!  I corrected the Line to:
```
notice("value is still ${non_sensitive.unwrap}")
```